Program.cs

```diff
@@ -1,57 +1,52 @@
 ﻿using System;
-using MySql.Data.MySqlClient;
+using System.Collections.Generic;
+using System.Linq;
+using Mercury.Models;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
 
 namespace Mercury
 {
     class Program
     {
 
-        public static void runQuery(string query)
-        {
-
-            if (query == "")
-            {
-                Console.WriteLine("please enter a query");
-                return;
-            }
-            else
-            {
-                string con = "datasource = 127.0.0.1;port = 3306 ; username=root;password=;database=mercury";
-                MySqlConnection c = new MySqlConnection(con);
-                MySqlCommand com = new MySqlCommand(query, c);
-                try
-                {
-                    c.Open();
-                    MySqlDataReader myreader = com.ExecuteReader();
-                    if (myreader.HasRows)
-                    {
-                        while (myreader.Read())
-                        {
-                            Console.WriteLine(myreader.GetString(0) + " - " + myreader.GetString(1) + " - " + myreader.GetString(2)
-                            + " - " + myreader.GetString(3) + " - " + myreader.GetString(4) + " - " + myreader.GetString(5));
-                        }
-                    }
-                    else
-                    {
-                        Console.WriteLine("query successfully excuated");
-                    }
-
-
-                }
-                catch (Exception e)
-                {
-                    Console.WriteLine("query error :" + e.Message);
-                }
-            }
-        }
 
         static void Main(string[] args)
         {
-            //  string myquery1 = "INSERT INTO USER VALUES (NULL , 'muhannad' ,'imad' , '123123' ,'moh@gmail.com','qalqilia' ,'0599123123') ";
-            string myquery2 = "SELECT * FROM USER";
-            runQuery(myquery2);
 
+           
+
+
+             // return raw by id : 
+        //    MercuryContext m =new MercuryContext();
+        //    var mp = m.Product.Single(b => b.Id.Equals(2));
+        //    Console.WriteLine(mp.Quantity);
+    
+
+
+          // add new entity :
+        // var p = new Product(){
+        //  Description = "bmw x5",
+        //  Price = 200000,
+        //  Quantity = 5
+        // };
+        // var m= new MercuryContext();
+        // m.Add(p);
+        // m.SaveChanges();
 
-        }
+
+
+     
+
+      var context = new MercuryContext();
+      
+      var x =   context.Product.Where(s => s.Id ==3).ToList();
```

Comment:
why not using the repo

Reply:
i forget to delete the repo file it is not a part of project , the repo folder use ADO.NET to manipulation database ,but we want to use entity framework to manipulate database .